types/knuddels-userapps-api/index.d.ts

```diff
@@ -11,2981 +11,3075 @@ export type KnuddelsEvent = string | Json | KnuddelsEventArray;
 
 // helper interfaces
 declare global {
-    interface Json {
-        [x: string]: JsonData | undefined;
-    }
+	interface Json {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/knuddels-userapps-api/index.d.ts;
  Line: 14;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.